app/api/V2/models/usermodels.py

```diff
@@ -1,36 +1,57 @@
 """COntain User model."""
```

Comment:
kindly fix the typos.

Reply:
Fixed the typo in the usermodels.py  header doctsring